spec/models/generic_object_definition_spec.rb

```diff
@@ -274,4 +274,102 @@
       expect(subject.first).to eq(@g1)
     end
   end
+
+  context 'property attribute name' do
```

Comment:
A good example to use `shared_examples_for` block since property attribute/associations/methods share almost the same group of tests.